onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -1605,7 +1605,8 @@ def aten_exp2(self: TFloat) -> TFloat:
 def aten_expand(self: TensorType, size: INT64) -> TensorType:
     # expand(Tensor(a) self, SymInt[] size, *, bool implicit=False) -> Tensor(a)
 
-    return op.Expand(self, size)
+    size_int64 = op.Cast(size, to=7)    # to INT64
```

Comment:
```suggestion
    size = op.Cast(size, to=INT64.dtype)
```

- fine to assign size, for consistency. 
- changed the magic number to INT64.dtype

Reply:
good idea